assets/src/scss/new-identity/style.scss

```diff
@@ -1,5 +1,16 @@
 @import "../base/tokens";
 
+@mixin shared-link-styles {
```

Comment:
@mleray I've created this `mixin` to share these styles through the other links.

Reply:
I prefer to keep it here and move to the `_mixins.scss` once the new identity is released.

What do you think?